app/js/views/camera-service.jsx

```diff
@@ -0,0 +1,90 @@
+/* global React */
```

Comment:
Here is example... depends on several foxbox PRs. I'll wrap up something very soon so that you can test this.